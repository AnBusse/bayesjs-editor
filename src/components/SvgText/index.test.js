import React from 'react';
import { shallow } from 'enzyme';
import SvgText from './component';

const defaultProps = {
  text: 'text',
  x: 5,
  y: 5,
  height: 20,
  width: 160,
};

const shallowComponent = (props = {}) => {
  const compProps = { ...defaultProps, ...props };

  return shallow(<SvgText {...compProps} />);
};

describe('SvgText Component', () => {
  it('matches snapshot', () => {
    const component = shallowComponent();

    expect(component).toMatchSnapshot();
  });
});
