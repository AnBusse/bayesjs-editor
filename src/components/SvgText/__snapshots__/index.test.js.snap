// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SvgText Component matches snapshot 1`] = `
<foreignObject
  data-testid="svg-text"
  height={20}
  width={160}
  x={5}
  y={5}
>
  <p
    style={
      Object {
        "margin": 0,
        "overflow": "hidden",
        "textOverflow": "ellipsis",
        "whiteSpace": "nowrap",
      }
    }
    title="text"
  >
    text
  </p>
</foreignObject>
`;
